src/metax_api/tests/api/rest/base/views/datasets/write.py

```diff
@@ -3809,3 +3811,332 @@ def test_user_projects_are_checked_when_writing_into_protected_data_catalog(self
         # try editing again - should be ok
         response = self.client.put('/rest/datasets/%d' % cr_data['id'], cr_data, format="json")
         self.assertEqual(response.status_code, status.HTTP_200_OK, response.content)
+
+class CatalogRecordApiWriteREMS(CatalogRecordApiWriteCommon):
+    rf = RDM.get_reference_data(cache)
+    permit_rights = {
+        # license type does not matter
+        "license": [
+            {
+                "title": rf['reference_data']['license'][0]['label'],
+                "identifier": rf['reference_data']['license'][0]['uri']
+            }
+        ],
+        "access_type": {
+            "in_scheme": "http://uri.suomi.fi/codelist/fairdata/access_type",
+            "identifier": "http://uri.suomi.fi/codelist/fairdata/access_type/code/permit",
+            "pref_label": {
+                "en": "User needs permit",
+                "fi": "Käyttäjän täytyy anoa lupaa käyttää aineistoa",
+                "und": "Käyttäjän täytyy anoa lupaa käyttää aineistoa"
+            }
+        }
+    }
```

Comment:
How come some ref data values are hardcoded, but above for license they are taken from ref data?